File: Data Structures & Algorithms/reverse-integer/submission-0.py
class Solution:
    def reverse(self, x: int) -> int:
        # print(x)
        q=x
        x=abs(x)
        l=(2**31)-1
        nega=False
        if q<0:
            nega=True
        r=0
        while x!=0:
            r=r*10+(x%10)
            x//=10
        if r>=l:
            return  0
        if nega:
            r*=-1
        return r

        

File: Data Structures & Algorithms/is-palindrome/submission-0.cs
public class Solution {
    public bool IsPalindrome(string s) {
        s=s.ToLower();
        string re="";
        foreach(char c in s){
            if(char.IsLetterOrDigit(c)){
                re+=c;
            }

        }
        string n=re.Replace(" ","");
        string rev=new string(re.Reverse().ToArray());
        if(rev.Equals(n)){
            return true;
        }
        return false;
    }
}
